jina/clients/__init__.py

```diff
@@ -17,7 +17,7 @@
 def Client(
     asyncio: Optional[bool] = False,
     host: Optional[str] = '0.0.0.0',
-    port_expose: Optional[int] = None,
+    port_gateway: Optional[int] = None,
```

Comment:
If `Client` always connects to gateway, should we just call it `port`?

Reply:
Hmm fair enough.